src/seed_tools/utils/study_validation.test.ts

```diff
@@ -383,12 +419,16 @@ describe('validateStudy', () => {
       ],
     });
 
-    expect(() => {
-      study_validation.validateStudy(study, studyFilePath);
-    }).toThrowError('Duplicate param name: test in experiment experiment1');
+    expect(
```

Comment:
any options to implement a helper like `expectHasError(study_validation.validateStudy(study, studyFilePath), 'error-string')`?

Reply:
will be replaced later with tests cleanup.